src/main/java/org/monarchinitiative/biodownload/BioDownloaderBuilder.java

```diff
@@ -0,0 +1,224 @@
+package org.monarchinitiative.biodownload;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.LinkedList;
+import java.util.List;
+
+public class BioDownloaderBuilder {
+
+    private static final Logger logger = LoggerFactory.getLogger(BioDownloaderBuilder.class);
+
+    private final List<DownloadableResource> resources;
+    private final Path destinationPath;
+    private boolean overwrite = false;
+
+
+    public BioDownloaderBuilder(String destinationPath) {
```

Comment:
I think it is better to make this method to accept `Path destinationPath`. Otherwise, the user may provide just any string and the `obodownload` needs to handle one more potential bug, while construction of a valid `Path destinationPath` in the client environment is not (I think) responsibility of `obodownload`.

Reply:
resolved in e89e320c